lib/ruby-mpd/song.rb

```diff
@@ -24,6 +24,18 @@ def ==(another)
     self.class == another.class && self.file == another.file
   end
 
+  def to_json(*a)
+    {
```

Comment:
Use symbols as keys, and do a `{ ... }.merge(@data)`. `data` should not be a nested structure.